### DISCUSSION TITLE
[핸즈온 랩 세션: 클라우드 아드보캇과 함께하는 블로그 사이트 만들기] 질문

### DISCUSSION THREAD

**Discussion by gradeten** [devrel-kr/HackaLearn#173] [3 Q&A - 일반] [answered]
<블로그 사이트 만들기> 라이브에서 알려주신대로 azurestaticapps.net 로 끝나는 사이트를 만들었는데, 배포된 사이트를 다른 팀원이 열거나, 제가 Visual Studio Code를 닫은 이후로 들어갔더니 글이 뜨지 않습니다. 왜 그런건지, 해결 방법은 무엇인지 궁금합니다!

VS Code를 닫지 않은 상태로 들어간 사이트 최초 모습: 
<img width="717" alt="나" src="https://user-images.githubusercontent.com/88281367/129229357-e2b8e4ad-9f14-4fef-95f7-9b6d8c3cba97.png">

팀원이 들어간 사이트 모습 / VS Code를 닫은 후 들어간 사이트 모습:
<img width="737" alt="팀원" src="https://user-images.githubusercontent.com/88281367/129229396-d43f26a7-c3d7-4d4a-92c0-96000bf29be4.png">

**Comment by justinyoo** [comment, marked as answer]
1. 비주얼 스튜디오 코드를 닫았단 얘기는 앱 실행까지도 종료됐을 가능성이 높습니다. 혹시 비주얼 스튜디오 코드 안의 터미널에서 `gatsby develop`, `func start`, `swa start http://localhost:8000 --api ./api` 명령어를 실행시키셨나요? 만약 그렇다면 비주얼 스튜디오 코드를 닫으면 이 앱들도 실행이 강제로 종료되기 때문에 그럴 수 있습니다. 웹 브라우저에서 F12 키를 눌러 개발자 모드를 활성화 한 후 콘솔 탭에서 어떤 에러가 나오는지 확인해 보시겠어요?

2. 마찬가지로 배포를 했을 경우에도 F12 키를 눌러 개발자 모드를 활성화 한 후 콘솔 탭에서 어떤 에러가 나오는지 확인해 보세요.

여기에 스샷을 남겨주시면 문제를 함께 해결하기가 수월합니다.

**Reply by gradeten** [reply]
답변 주셔서 감사합니다! console창 에러 스크린샷 올립니다.
<img width="1440" alt="콘솔 에러" src="https://user-images.githubusercontent.com/88281367/129355118-ec2b45ac-aa5a-4c11-919e-10f6f62ad1c3.png">

**Reply by justinyoo** [reply]
고맙습니다! 이 웹사이트가 로컬에서 돌린 것인지, 배포한 사이트에서 돌린 것인지 알 수가 없는데요, 관련 정보를 같이 알려주셔야 할 것 같아요.

일단, 콘솔 에러를 놓고 보면:

1. 첫번째 에러는 현재 http://localhost:7071/api/posts 엔드포인트에 접근할 수 없습니다. 이는 로컬에서 애저 펑션 앱이 돌고 있지 않다는 것을 의미해요.
2. 두번째 에러는 마찬가지로 `fetch()` 함수를 통해 API 결과값을 가져올 수 없다는 것을 의미합니다. 이건 배포한 사이트 기준인 것 같은데 지금... 두가지를 한꺼번에 해결하려면 힘드니 한번에 하나씩 해 볼까요?
